gensim/test/test_utils.py

```diff
@@ -259,6 +263,46 @@ def test_save_as_line_sentence_ru(self):
             self.assertEqual(sentences, ref_sentences)
 
 
+class HashTest(unittest.TestCase):
+    def setUp(self):
+        self.expected = {
+            u'команда': 1725507386,
+            u'маленьких': 3011324125,
+            u'друзей': 737001801,
+            u'возит': 4225261911,
+            u'грузы': 1301826944,
+            u'всех': 706328732,
+            u'быстрей': 1379730754
+        }
+        self.expected_broken = {
+            u'команда': 962806708,
+            u'маленьких': 3633597485,
+            u'друзей': 214728041,
+            u'возит': 3590926132,
+            u'грузы': 3674544745,
+            u'всех': 3931012458,
+            u'быстрей': 822471432,
+        }
+
+    def test_python(self):
+        actual = {k: gensim.models.utils_any2vec._ft_hash_py(k) for k in self.expected}
+        self.assertEqual(self.expected, actual)
+
+    @unittest.skipIf(DISABLE_CYTHON_TESTS, 'Cython tests are currently disabled')
```

Comment:
"Cython hash functions aren't properly compiled"  - this is only one reason why this can be disabled (here and everywhere)